app/assets/javascripts/discourse/app/components/user-menu/reviewables-list.js

```diff
@@ -0,0 +1,31 @@
+import UserMenuItemsList from "discourse/components/user-menu/items-list";
+import { ajax } from "discourse/lib/ajax";
+import { MiniReviewable } from "discourse/models/reviewable";
+import I18n from "I18n";
+import getUrl from "discourse-common/lib/get-url";
+
+export default class UserMenuReviewablesList extends UserMenuItemsList {
+  get showAll() {
+    return true;
+  }
```

Comment:
I wonder if this getter is really necessary. I think we can rely on the assumption that if `showAllHref` is defined, we should render the link.